UndertaleModLib/UndertaleChunks.cs

```diff
@@ -1576,11 +1576,15 @@ internal override void SerializeChunk(UndertaleWriter writer)
         // GMS 2.0.6.96 is the oldest available runtime version,
         // so this actually could be some other version between GMS 2.0 - 2.0.6.
         // (the oldest copy of "Zeus-Runtime.rss" on web.archive.org has this version as first one)
+        private static bool checkedFor2_0_6;
```

Comment:
this bool should probably be located where all the other attributes are instead of right before t his function.